store-implementation/accumulo-store/src/test/resources/store.properties

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-gaffer.store.class=uk.gov.gchq.gaffer.accumulostore.SingleUseMockAccumuloStore
-accumulo.instance=byteEntityInstance
-accumulo.zookeepers=aZookeeper
+gaffer.store.class=uk.gov.gchq.gaffer.accumulostore.SingleUseMiniAccumuloStore
+accumulo.instance=accumuloInstance
+accumulo.zookeepers=zookeeper
```

Comment:
don't you need to use localhost?

Reply:
zookeeper is updated in the properties with the actual value when MiniAccumuloClusterManager starts the cluster.
See above.

Reply:
Thanks for clarifying. Would it be possible to remove them or add a comment above it explaining that it get's overwritten?

Reply:
Comment added to the file.